ui/app/components/clients/activity.ts

```diff
@@ -10,15 +11,16 @@ import Component from '@glimmer/component';
 import { isSameMonth, fromUnixTime } from 'date-fns';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
 import { calculateAverage } from 'vault/utils/chart-helpers';
-import { filterVersionHistory } from 'core/utils/client-count-utils';
+import { filterVersionHistory, hasMountsKey, hasNamespacesKey } from 'core/utils/client-count-utils';
 
 import type ClientsActivityModel from 'vault/models/clients/activity';
-import type {
-  ClientActivityNewClients,
```

Comment:
I was getting a little jumbled up with all of the types named `ClientActivity...` so renamed them to be a little more explicit with which part of the response they're a declaration for